cmd/kube-apiserver/app/options/plugins.go

```diff
@@ -52,15 +51,22 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/admission/securitycontext/scdeny"
 	"k8s.io/kubernetes/plugin/pkg/admission/serviceaccount"
 	"k8s.io/kubernetes/plugin/pkg/admission/storageclass/setdefault"
+
+	"k8s.io/apiserver/pkg/admission/plugin/initialization"
+	"k8s.io/apiserver/pkg/admission/plugin/namespace/lifecycle"
+	mutatingwebhook "k8s.io/apiserver/pkg/admission/plugin/webhook/mutating"
+	validatingwebhook "k8s.io/apiserver/pkg/admission/plugin/webhook/validating"
+	genericoptions "k8s.io/apiserver/pkg/server/options"
 )
 
 // RegisterAllAdmissionPlugins registers all admission plugins
-func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
-	admit.Register(plugins)
+func RegisterAllAdmissionPlugins(admission *genericoptions.AdmissionOptions) {
```

Comment:
nit: I would add a comment saying that the method also sets the recommended plugins order.

Reply:
make sense